src/mlpack/methods/hoeffding_trees/hoeffding_tree_impl.hpp

```diff
@@ -500,7 +501,28 @@ size_t HoeffdingTree<
   else
     return 0; // Not sure what to do here...
 }
-
+template<typename FitnessFunction,
+         template<typename> class NumericSplitType,
+         template<typename> class CategoricalSplitType>
+size_t HoeffdingTree<
+    FitnessFunction,
+    NumericSplitType,
+    CategoricalSplitType
+>::NumDescendants()
+{
+  size_t nodes = 0;
+  std::stack<HoeffdingTree*> stack;
+  stack.push(this); // push the current tree
```

Comment:
Minor style issue, can you take a look at: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#comments.
Basically if you do: ``Push the current tree.`` it's fine.